README.md

```diff
@@ -143,32 +147,49 @@ defmodule YourApp.Router do
   use Phoenix.Router
 
   use BoomNotifier,
-    notification_trigger: [exponential: [limit: 100]]
+    count: [exponential: [limit: 100]]
     notifiers: [
       # ...
     ]
 ```
 
-### Notification trigger time limit
+### Time Based Throttle
+
+```elixir
+defmodule YourApp.Router do
+  use Phoenix.Router
+
+  use BoomNotifier,
+    notifiers: [
+        # ...
+      ],
+    groupping: :time,
```

Comment:
Does it makes sense to have `:time` as other `notification_trigger` strategy instead of adding a new option? And maybe we could use the same time limit key we use for the others